build_tarballs.jl

```diff
@@ -147,8 +156,15 @@ if [[ "${target}" == *apple* ]] || [[ "${target}" == *freebsd* ]]; then
     export ac_cv_have_decl___builtin_ffs=yes
 fi
 
+if [[ "${target}" == *mingw* ]]; then
+    CMAKE_CPP_FLAGS="${CMAKE_CPP_FLAGS} -remap -D__USING_SJLJ_EXCEPTIONS__ -D__CRT__NO_INLINE"
+    # The joy of Windows
+    echo "BaseTsd.h basetsd.h" >> /opt/${target}/${target}/include/header.gcc
```

Comment:
Can you explain a bit of what this is doing? Should this be done as part of the base image for all builds?